element-hq/element-x-android
# Alert for incoming call even if notifications are disabled - WAITING FOR FINAL PRODUCT DECISION

## Type of change


## Content



Alert for incoming call even if notifications are disabled.

Also change notification color to `FF0DBD8B`:

|Before|After|
|-|-|
|<img width="500" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/c494a7b5-885c-400e-b06f-2f1ec64cecfd">|<img width="500" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/ca50cbcf-06c4-48c9-b44b-0f6edd8d18d0">|

Other commits are cleanup.

## Motivation and context


Ensure that when the user has disabled the notification, they will still receive alert for incoming call, which is a separate feature.

User still have the ability to disable incoming call by tweaking the notification channel settings.

Closes #3055

## Screenshots / GIFs



## Tests



- Disable notification in the app settings.
- Receive a Call
- Observe that the device is alerting for the incoming call. Previously it was not the case.

## Tested devices

- OS version(s):

## Checklist
